ReactWindows/ReactNative/Views/DatePicker/ReactCalendarDatePickerShadowNode.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Facebook.Yoga;
+using ReactNative.UIManager;
+
+namespace ReactNative.Views.DatePicker
+{
+    /// <summary>
+    /// The shadow node implementation for <see cref="ReactNative.Views.DatePicker.ReactCalendarDatePickerManager" /> views.
+    /// </summary>
+    public class ReactCalendarDatePickerShadowNode : LayoutShadowNode
+    {
+        /// <summary>
+        /// Instantiates the <see cref="ReactDatePickerShadowNode"/>.
+        /// </summary>
+        public ReactCalendarDatePickerShadowNode()
+        {
+            MeasureFunction = MeasurePicker;
+        }
+
+        private static YogaSize MeasurePicker(YogaNode node, float width, YogaMeasureMode widthMode, float height, YogaMeasureMode heightMode)
+        {
+            return MeasureOutput.Make(width, 40f);
+        }
```

Comment:
I believe the height is 32, you may also need to check if the height is undefined, what happens if the user specifies the height in JavaScript?